### PR TITLE
Add launch stage to library tooltip

### LINKED ISSUE #2770: Indicate status of cloud libraries as GA, alpha, beta, or deprecated
Perhaps in a tooltip? Or by splitting into separate, perhaps collapsible sections?

### PR DESCRIPTION
**Pull request by elharo**
@chanseokoh fix #2770

### CODE REVIEW

File: gcloud-eclipse-tools.launch [hunk 1]
```diff
@@ -6,7 +6,7 @@
 <booleanAttribute key="automaticValidate" value="false"/>
 <stringAttribute key="bad_container_name" value="/trunk/gcloud-eclipse-tools-launcher"/>
 <stringAttribute key="bootstrap" value=""/>
-<stringAttribute key="checked" value="[NONE]"/>
+<stringAttribute key="checked" value="com.google.cloud.tools.eclipse.sdk"/>
```
Comment (chanseokoh) [inline gcloud-eclipse-tools.launch]: Revert the tracing change to the launch config.

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/LibrarySelectorGroup.java [hunk 2]
```diff
@@ -89,8 +89,15 @@ private void createContents(Composite parentContainer, String groupLabel) {
     for (Library library : availableLibraries.values()) {
       Button libraryButton = new Button(apiGroup, SWT.CHECK);
       libraryButton.setText(getLibraryName(library));
-      if (library.getToolTip() != null) {
-        libraryButton.setToolTipText(library.getToolTip());
+      String toolTip = library.getToolTip();
+      
+      String stage = library.getLaunchStage();
+      if (!"GA".equals(stage )) {
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/LibrarySelectorGroup.java]: code style: `stage)) {`.

File: gcloud-eclipse-tools.launch (hunk 1 above)
Comment (elharo) [reply gcloud-eclipse-tools.launch]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/LibrarySelectorGroup.java (hunk 2 above)
Comment (elharo) [reply plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/LibrarySelectorGroup.java]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/Library.java [hunk 3]
```diff
@@ -67,6 +68,10 @@ public String getName() {
   public void setName(String name) {
     this.name = name;
   }
+
+  public String getLaunchStage() {
+    return this.stage;
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/Library.java]: I don't think we usually use `this.`?

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/LibrarySelectorGroup.java [hunk 4]
```diff
@@ -89,8 +89,15 @@ private void createContents(Composite parentContainer, String groupLabel) {
     for (Library library : availableLibraries.values()) {
       Button libraryButton = new Button(apiGroup, SWT.CHECK);
       libraryButton.setText(getLibraryName(library));
-      if (library.getToolTip() != null) {
-        libraryButton.setToolTipText(library.getToolTip());
+      String toolTip = library.getToolTip();
+      
+      String stage = library.getLaunchStage();
+      if (!"GA".equals(stage)) { //$NON-NLS-1
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/LibrarySelectorGroup.java]: The code previous guarded against the tooltip being `null`.

File: gcloud-eclipse-tools.launch (hunk 1 above)
Comment (briandealwis) [reply gcloud-eclipse-tools.launch]: This hasn't been pushed up?

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/LibrarySelectorGroup.java (hunk 4 above)
Comment (elharo) [reply plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/LibrarySelectorGroup.java]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/Library.java (hunk 3 above)
Comment (elharo) [reply plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/Library.java]: done

**Pull request merged**